### ISSUE TITLE
Page Connectivity failed

### ISSUE THREAD

**Issue by Trustedbrands** [inspireui/fluxbuilder#186]
Hi, I'm getting this error message with the new FluxBuilder 1.0.7
![Page connectivity](https://user-images.githubusercontent.com/69647853/90226629-8e37fc00-de13-11ea-9d5e-619d94b4e645.PNG)

**Comment by Trustedbrands** [comment]
If I disable the Woodmart core plugin the problem goes away. But then the website does not work.

**Comment by minhcasi** [comment]
@Trustedbrands - could you able to display the product and category from Data review section? Thank you!

**Comment by Trustedbrands** [comment]
@minhcasi 
Yes, I got the product and category data from Data review section

**Comment by Trustedbrands** [comment]
@minhcasi 
It seems that the issue has been fixed with the new update of the Mstore plugin.
But now Fluxbuilder on Mac is broken. Try to add a new App, performs analysis, but everything stops there.

Trying to activate the license in Build, but none of the fields can be edited.

**Comment by minhcasi** [comment]
@Trustedbrands - would you mind to share your consumse/serect keys on your website - we will help to investigate the issue as it seems working fine with our testing.

You could send privately to inspireui.team@gmail.com - thank you!

**Comment by Trustedbrands** [comment]
@minhcasi Sure!

**Comment by minhcasi** [comment]
@Trustedbrands - sorry we did not get your email - could i know your email address to check from our inbox -thanks

**Comment by Trustedbrands** [comment]
@minhcasi I sent you an email from patrick@trustedbrands.io

**Comment by Trustedbrands** [comment]
@minhcasi  Now I have a problem that I do not get categories that belong to my website, only categories from your demo come up.
![Cant see my category](https://user-images.githubusercontent.com/69647853/92187207-71fb0e00-ee59-11ea-8c2c-d808cf7df973.PNG)
![Catgory](https://user-images.githubusercontent.com/69647853/92187211-732c3b00-ee59-11ea-91c5-e589ae401cf0.PNG)

**Comment by Trustedbrands** [comment]
@minhcasi @dttrung26 @vumanhsn97  How can I view recent products? Looks like I need to select a specific category for the widget to save. But i want to show news from all categories.
It also looks like the vertical layout widget is missing from the add widget and it is not possible to drag to change the order of the widget
